data/human/transport missions.txt

```diff
@@ -3582,3 +3582,714 @@ mission "Hauler VI cargo prototype 3"
+			`	See waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the hanger shortly."`
+			`	You go back to your ships hanger and sure enough there are three people with backpacks waiting for you with a few cases. They introduce themselves as Coya, River and Bunker.`
+			choice
+				`	"I am Captain <first> <last>."`
+				`	"I'm <first>."`
+					goto loadup
+			`	They all grimace at your formality.`
+			label loadup
+			`	"Good. Now that that's done with let's load up," says Bunker. "Taking down the man. Wwhhoo! Love it."`
+				accept
+			label imout
+			`	"I'm sorry to hear that," she says between gulps of her smoothie. "Well I suppose it just wasn't meant to be," she finishes her "drink" with lightning speed, gets up and leaves.`
+				decline
+	on visit
+		dialog `The filming equipment is aboard an escort. Better take off and wait for it to arrive.`
+
+#######################################################################################################################
+
+mission "Journalism 3"
+	landing
+	name "Slave trade"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction."
+	minor
+	source "Maker"
+	destination "Hippocrates"
+	to offer
+		has "Journalism 2: done"
+	passengers 1
+	cargo "stealth drones" 1
+	on offer
+		conversation
+			`You drop off Ale, Coya and River at the spaceport and wish them luck. They are visibly embarrassed to be in security uniforms or perhaps uncomfortable with their disguises which make them virtually unrecognizable. Bunker can't stop laughing. Everyone exchanges wishes of good luck and they walk off into the spaceport. You and bunker go back to the <ship> and prepare to launch.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+####################################################################################################################
+
+mission "Journalism 4"
+	infiltrating
+	landing
+	name "Slave trade"
+	description "Pick up the team on <destination>."
+	minor
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Journalism 3: done"
+	passengers 1
+	cargo "stealth drones" 1
+
+	on offer
+		conversation
+			`It's not hard to find a remote canyon within range of the target area. You stay hovering in case you need a quick escape but are well inside the atmosphere so Bunker releases the drones from the open cargo bay without a pressure suit. They are so small you can't make them out from the cockpit, nor can you see them on your ships scanners. When Bunker returns he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`
```

Comment:
```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but as you are well within the atmosphere Bunker releases the drones from the open cargo bay without needing a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`
```

Reply:
I'd recommend changing `the <ship>` to just `your ship`. Trying to anticipate what players will name ships and making that fit in the writing without causing grammatical problem is tough.